web/source/text/kbdInterface.ts

```diff
@@ -861,11 +843,6 @@ namespace com.keyman.text {
       // Automatically manages affected deadkey positions.
       outputTarget.insertTextBeforeCaret(s);
       outputTarget.restoreProperties();
-
-      // Refresh element content after change (if needed)
-      if(typeof(keyman.refreshElementContent) == 'function') {
```

Comment:
Can you explain the move of `refreshElementContent` call?

Reply:
It's basically the same as `doInputEvent` (for DOM-based web pages), but instead oriented to notify the embedding app.

We really shouldn't be signalling the app every time we produce an 'alternate' (fat-finger guess) for predictive text, but we currently are.  Moving it as this PR does ensures we only signal the app for the actual output sequence.